### ISSUE TITLE
Updating the glossary of activation functions

### ISSUE THREAD

**Issue by PhilipNaveen** [bfortuner/ml-glossary#132]
Currently the glossary of activation functions is limited to only a few functions. There are many newer functions such as Swish, Mish, Phish, Softplus, GELU, etc. that are missing from the glossary.

**Comment by bfortuner** [comment]
Yeah! I'm so behind on activation functions. Do you have any interested in contributing? :)

**Comment by PhilipNaveen** [comment]
I would definitely be interested on contributing!

**Comment by saydemr** [comment]
Providing maxout and parametric ReLU would be great. Any progress on the others?
